CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] OTLP: Change otlp handler to consist with the Prometheus otlp handler. Enable `target_info` metric and disable convert all attributes to labels by default. Add a config to specify promote resource attributes for tenants. #6272
```

Comment:
`Change OTLP handler to be consistent with the Prometheus OTLP handler`.

How about we do it this way:

```
* [CHANGE] OTLP: Change OTLP handler to be consistent with the Prometheus OTLP handler
 - `target_info` metric is enabled by default and can be disabled via `xxx` flag
 - `Convert all attributes to labels are disabled by default and can be enabled via `xxx` flag
 - Added config name `xxx` to specify promote ...
```